cinema/permissions.py

```diff
@@ -0,0 +1,9 @@
+from rest_framework.permissions import BasePermission, SAFE_METHODS
```

Comment:
The import statement is correct. `SAFE_METHODS` is used to check if the request method is read-only, which is appropriate for this permission class.